config/initializers/rspec_api_documentation.rb

```diff
@@ -0,0 +1,100 @@
+module RspecApiDocumentation
```

Comment:
Shouldn't this be on the same unless block below?

Reply:
Sure! and it's a nice idea to override that tokens indeed, I will add it.